cpp/splinepy/fitting/fitting.hpp

```diff
@@ -36,4 +36,16 @@ double ApproximateCurve(const double* points,
                         std::vector<double>& coefficient_matrix,
                         std::vector<double>& control_points);
 
+void ApproximateSurface(const double* points,
```

Comment:
Please make this doxygenable (vscode supports it if you just type /** <enter>)

Reply:
Sorry, I dont understand this feedback, can you clarify?